package.json

```diff
@@ -36,29 +36,28 @@
 		"prepare": "npm run format && npm run build"
 	},
 	"peerDependencies": {
-		"@nestjs/common": "^7.4.0 || ^8.1.2",
-		"@nestjs/microservices": "^7.4.0 || ^8.1.2"
+		"@nestjs/common": "^7.4.0 || ^8.1.2 || ^10.3.3",
```

Comment:
If you're adding support for 10 should 9 also be added to ensure users with 9 can also leverage the lib?

Reply:
We have an immediate need to get it working with nest 10. I don't see why this couldn't work with nest 9, but I don't know what the compatible versions are of each of the dependencies. 

If you have a specific version of nest 9 you want to use this with, I can test against it to be sure